data/wsl/create_unattend_raw.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Check for correct arguments
+if [ "$1" != "BIOS" ] && [ "$1" != "UEFI" ] && [ "$1" != "ARM64" ]; then
+    echo "You should declare UEFI, BIOS or ARM64!"
+    echo "  USAGE: ${0} <BIOS/UEFI/ARM64>"
+    exit 1
+fi
+
+# Check if the Autounattend file exists
```

Comment:
I believe `unattended` is the proper term here. :)

Reply:
Autounattend.xml is how MS calls the files :shrug: